lib/datadog/tracing/span.rb

```diff
@@ -106,6 +107,12 @@ def initialize(
         # duration_start and duration_end track monotonic clock, and may remain nil in cases where it
         # is known that we have to use wall clock to measure duration.
         @duration = duration
+
+        # https://docs.datadoghq.com/tracing/visualization/#service-entry-span
+        # A span is a service entry span when it is the entrypoint method for a request to a service.
+        # You can visualize this within Datadog APM when the color of the immediate parent on a flame graph is a different
+        # color. Services are also listed on the right when viewing a flame graph.
+        @service_entry = service_entry
```

Comment:
Can you move this documentation to `def service_entry?`, that way it will be visible in our public YARD docs.